### PR TITLE
Pom 275 120 captcha

### PR DESCRIPTION
**Pull request by klimtron**

### CODE REVIEW

File: src/app/core/translations/pl_PL.ts [hunk 1]
```diff
@@ -155,4 +155,6 @@ export default {
   NO_RESULTS_TEXT_START: 'Zmień kryteria wyszukiwania, lub',
   NO_RESULTS_TEXT_LINK: 'przejdź do informatora',
   NO_RESULTS_TEXT_END: 'by wyświetlić inne dostępne formy pomocy dla tej kategorii.',
+  MESSAGE_SENT_ALERT_HEADER: 'Twoja odpowiedź została wysłana do osoby pomagającej',
```
Comment (maciejp-github) [inline src/app/core/translations/pl_PL.ts:158]: Missing dot at the end of the sentence, is it intended?

File: src/app/find-help/reply-offer/reply-offer.component.ts [hunk 2]
```diff
@@ -35,6 +41,22 @@ export class ReplyOfferComponent implements OnInit {
...
 
   submitMessage(): void {
-    this.messageResourceService.sendMessageMessage(this.data).subscribe((response) => {});
+    this.loading = true;
+
+    this.reCaptchaV3Service
+      .execute('sendMessageMessage')
+      .pipe(
+        switchMap((captchaToken) => {
+          return this.messageResourceService.sendMessageMessage(this.data, captchaToken);
```
Comment (broslukasz) [inline src/app/find-help/reply-offer/reply-offer.component.ts:50]: sendMessageMessage ;)

**Review by broslukasz — APPROVED** [review]

File: src/app/core/translations/pl_PL.ts (hunk 1 above)
Comment (mgol) [reply src/app/core/translations/pl_PL.ts:164]: @maciejp-github This is a header so maybe yes?

File: src/app/find-help/reply-offer/reply-offer.component.ts (hunk 2 above)
Comment (klimtron) [reply src/app/find-help/reply-offer/reply-offer.component.ts:50]: jak w serwisie ;) 
image

**Pull request merged**